frontend/src/components/survey-editor/QuestionGroupEditor.jsx

```diff
@@ -379,28 +379,6 @@ const QuestionGroupEditor = ({ index, questionGroup, isMoving }) => {
     });
   };
 
-  const checkQuestionOptionsAndRepeatingObjects = (sources) => {
```

Comment:
Hi @navins94 , 
Why was this function deleted?
And I think some last updated on develop reverted by your updates.

Reply:
I think it got deleted while merging from develop branch. Let me look into it